tool.go

```diff
@@ -771,7 +771,7 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 		allArgs = []string{"--require", "source-map-support/register"}
 		if err := exec.Command("node", "--require", "source-map-support/register", "--eval", "").Run(); err != nil {
 			if !quiet {
-				fmt.Fprintln(os.Stderr, "gopherjs: Source maps disabled. Use Node.js 4.x with source-map-support module for nice stack traces.")
+				fmt.Fprintln(os.Stderr, "gopherjs: Source maps disabled. Install source-map-support module for nice stack traces. See https://github.com/gopherjs/gopherjs#installation-and-usage.")
```

Comment:
I wonder if it would also be better to link directly to the syscalls page (https://github.com/gopherjs/gopherjs/blob/master/doc/syscalls.md)?  Or make it more prominent on the page linked to (possibly with an anchor tag)?